Resources/public/css/components/forms.css

```diff
@@ -0,0 +1,30 @@
+/*
+ * AJAX loading states
+ * ----------------------------------------------------------------------------------------------------------------- */
+
+.field-short-description.loading {
+    /* center the spinner in association field descriptions */
+    text-align: center;
+}
+
+/*
+ * Sortable associations
+ * ----------------------------------------------------------------------------------------------------------------- */
+
+.sonata-ba-sortable-handler {
+    cursor: grab;
+}
+.sonata-ba-sortable td.sonata-ba-sortable-handler {
+    text-align: center;
+}
+
+/* OMG, AdminLTE, the specificity ! */
+.nav-tabs-custom > .nav-tabs > li.has-error > a {
+    color: #a94442; /* $text-danger */
+}
+.nav-tabs .fa {
+    display: none;
+}
+.nav-tabs .has-error .fa {
+    display: inline-block;
+}
```

Comment:
Please configure your editor to add a newline at end of file (see that little red pictogram? That's what it means).